ui/static/ui/js/learning_resources.jsx

```diff
@@ -267,7 +267,10 @@ define('learning_resources',
       var vocabSelector = "There are no terms for this resource";
       var termSelector = "";
       var termList = "";
-
+      var spinnerClassName = "fa fa-spinner fa-pulse fa-3x lr-spinner-hide";
+      if (this.state.showSpinner) {
```

Comment:
Instead of setting a different CSS class please just set the spinner to a variable here where it is null if `showSpinner` is false and set to `<i className=... />` if it's true.